duckduckgo/chrome-zeroclickinfo
# firefox option page listener

**Reviewer:**



## Description:


1. Update the Firefox option page link to use messaging. 
2. Fix new tab popup domain.

## Steps to test this PR:

1. load in firefox as a webextension
2. check that option page link in the popup works
3. the popup should be greyed out for new tab and about pages


## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
